test/functional/c/test_symmetry.cpp

```diff
@@ -21,14 +21,15 @@ TEST(Symmetry, test_get_lattice_symmetry_layer) {
     };
     double positions[1][3] = {{0, 0, 0}};
     const int types[1] = {0};
-    const int aperiodic_axis = 2;
     const double symprec = 1e-5;
     const double angle_tolerance = -1;
 
     cell = cel_alloc_cell(size, NOSPIN);
     ASSERT_NE(cell, nullptr);
     ASSERT_EQ(spg_get_error_code(), SPGLIB_SUCCESS);
-    cel_set_layer_cell(cell, lattice, positions, types, aperiodic_axis);
+    for (int aperiodic_axis = 0; aperiodic_axis < 3; aperiodic_axis++) {
```

Comment:
Sorry for back and forth. Can you go back to use only `aperiodic_axis=2` or loop over until `symmetry = sym_get_operation(cell, symprec, angle_tolerance);`? Actually, looping over only `cel_set_layer_cell` does nothing.

Reply:
Sorry, I was too careless. I have modified it. Now Codecov reports that those lines have been covered ([`10f2bdc`](https://app.codecov.io/gh/spglib/spglib/commit/10f2bdcd7deaf27bfdf3506149d42c2bcc1dd269/blob/src/symmetry.c)).

Reply:
No worries! Thank you for taking part in layer-group implementation. It's tough work.